edxval/models.py

```diff
@@ -190,3 +194,16 @@ def content_type(self):
             return 'application/json'
         else:
             return 'text/plain'
+
+
+@receiver(models.signals.post_save, sender=Video)
+def video_status_update_callback(sender, **kwargs):  # pylint: disable=unused-argument
+    """
+    Log video status for an existing video instance
+    """
+    video = kwargs['instance']
+
+    if kwargs['created']:
+        logger.info('Video created with id [%s] and status [%s]', video.edx_video_id, video.status)
```

Comment:
Nit: I find it very useful to have a common prefix for all Splunk logs within a module.  It then makes it easier to query in Splunk.  So, for example, for VAL, you may consider prefixing all log messages with "VAL: "